clang/test/CIR/Lowering/builtin-binary-fp2fp.c

```diff
@@ -0,0 +1,132 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-llvm %s -o %t.ll
+// RUN: FileCheck --input-file=%t.ll %s -check-prefix=LLVM
```

Comment:
Nice. I think it's best to have it all in one place, since it's the exact same functions. Just add these RUN lines in `clang/test/CIR/CodeGen/builtin-floating-point.c` and add the `LLVM` checks there. No need to add tests for things that pre-exist in that file, we can increment that in the future.

Reply:
Well, LLVM lowering for the unary fp2fp builtins is not yet implemented. I'll make a PR for this later and we can add tests in that PR :)

Reply:
Ok. Looking fwd, I'm in general (unless there's a good reason not to) expecting new CIR stuff to come with LLVM support in the same PR. Just a heads up so you can make sure you can include that in newer PRs you create. If that potentially means bigger PRs, just do less CIR features in each of them.